src/test/java/com/google/api/codegen/testdata/go_example_library.baseline

```diff
@@ -20,7 +20,7 @@ package library_test
 import (
     "cloud.google.com/go/library/apiv1"
     "golang.org/x/net/context"
-    googleapis_example_library_v1 "google.golang.org/genproto/googleapis/example/library/v1"
+    pb "google.golang.org/genproto/googleapis/example/library/v1"
```

Comment:
Was the `pb` name a suggestion by the Go team?

I'd generally prefer `library` or `library_v1`... I know there's a tendency for very short names, but I feel `pb` has lost most of the context behind the import. My first thought was the `google/protobuf` package.